grunt-tasks/grunt-deps.js

```diff
@@ -0,0 +1,159 @@
+/* eslint-disable */
+var Graph = require('graphs');
+var esprima = require('esprima');
+var path = require('path');
+
+module.exports = function gruntDeps(grunt) {
+  grunt.config('deps', {
+    files: '../src/**/*.js',
+    cwd: './build',
+    template: 'release.tmpl',
+    output: 'release.jsb2',
+    modules: [{
+      name: 'crm',
+      location: '../src'
+    }]
+  });
+
+  grunt.registerTask('deps', function() {
+    var config = grunt.config.get('deps');
+    grunt.file.setBase(config.cwd);
+    var files = grunt.file.expand(config.files);
+    var graph = new Graph();
+    var nodes = {};
+
+    // Resolves import modules into a relative file path
+    function resolvePath(module, sourceFile) {
+      var config = grunt.config.get('deps');
+      var relative = module[0] === '.';
+      if (relative) {
+        var sourceDir = path.dirname(sourceFile);
+        return path.join(sourceDir, module) + '.js';
+      } else {
+        var parts = module.split('/');
+        var moduleName = parts.shift();
+        var config = config.modules.filter(function(m) {
+          return m.name === moduleName;
+        })[0];
+        var location = config && config.location;
```

Comment:
Same as above. This time the config.location gives understanding to the check already.